src/CRM/CivixBundle/Command/InitCommand.php

```diff
@@ -32,11 +32,13 @@ protected function configure() {
         "Create a new CiviCRM Module-Extension (Regenerate module.civix.php if ext.name not specified)\n" .
         "\n" .
         "<comment>Identification:</comment>\n" .
-        "  Keys must be lowercase alphanumeric (with dashes and underscores allowed).\n" .
+        "  Keys should be lowercase alphanumeric with underscores. Additionally, dots and dashes\n" .
+        "  are also legal - but they are discouraged for new extensions.\n" .
         "\n" .
-        "  Optionally, you may use a Java-style prefix (reverse domain name).\n" .
+        "  Historically, the key often included a Java-style prefix (reverse domain name).\n" .
+        "  The benefit of the prefix was mostly cosmetic. It did not provide consistent\n" .
+        "  isolation between similarly named extensions. It is now recommended to skip\n" .
```

Comment:
Is there some text missing after this? I get this from civix help generate:module

```
Help:
  Create a new CiviCRM Module-Extension (Regenerate module.civix.php if ext.name not specified)

  Identification:
    Keys should be lowercase alphanumeric with underscores. Additionally, dots and dashes
    are also legal - but they are discouraged for new extensions.

    Historically, the key often included a Java-style prefix (reverse domain name).
    The benefit of the prefix was mostly cosmetic. It did not provide consistent
    isolation between similarly named extensions. It is now recommended to skip


  Examples:
    civix generate:module foo-bar
    civix generate:module foo-bar --license=AGPL-3.0 --author="Alice" --email="alice@example.org"
    civix generate:module org.example.foo-bar
```